testsuites/rio/src/main/java/org/eclipse/rdf4j/rio/AbstractParserHandlingTest.java

```diff
@@ -948,4 +951,30 @@ private final Model getTestModel(String languageValue, String languageTag) {
 				vf.createLiteral(languageValue, languageTag)));
 		return result;
 	}
-}
+//
+//	@Test(expected = RDFParseException.class)
+//	public void testNullRdfHandlerReader() throws IOException {
+//
+//		getParser().parse(new InputStreamReader(new ByteArrayInputStream("".getBytes())) , "");
+//
+//	}
+
+	@Test
+	public void testNullRdfHandlerReader(){
+		try {
+			 getParser().parse(new InputStreamReader(new ByteArrayInputStream("".getBytes())), "");
```

Comment:
force the rdfhandler to be null. Maybe you can store getParser() in a variable and call something like .setHandler(null).